fahrschein/src/main/java/org/zalando/fahrschein/ManagedCursorManager.java

```diff
@@ -143,7 +144,7 @@ public void onSuccess(String eventName, Cursor cursor) throws IOException, Curso
     }
 
     private String inputStreamToString(InputStream inputStream) {
-        return new BufferedReader(new InputStreamReader(inputStream))
+        return new BufferedReader(new InputStreamReader(inputStream, StandardCharsets.UTF_8))
```

Comment:
a) this method is called in case of errors, so maybe not critical that we default to UTF-8 here instead of the actual charset used b) the charset in the content-type response header may or may not be present for indicating the encoding. It's not something we can easily introspect here, at least as of now.

What do you mean with the nio import should not be used?

Reply:
one thing is for sure: using the process default encoding (what the code was doing before) is probably worse than fixing this to UTF-8 at this line of code.